Infrastructure/Models/Address.cs

```diff
@@ -3,10 +3,14 @@
     public class Address
     {
         public int Id { get; set; }
+        public string Name { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
         public string City { get; set; }
         public string AddressLine1 { get; set; }
         public string? AddressLine2 { get; set; }
         public string PostalCode { get; set; }
```

Comment:
@ciliakas I think it is time to decide whether we will be showing pins on the map for 
a) Receiver address;
b) Address according to the status (don't like this personally)
c) Only Terminal addresses